{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 28
        },
        "etr": "2020-12-16T20:45:00-0400",
        "start": "2020-12-16T16:22:00-0400"
      },
      "geom": {
        "a": [
          "kfzoGrbm_KSdHaq@|y@{NlH}MoEnpAafA"
        ],
        "p": [
          "cp{oG`pn_K"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
